src/components/EditCoursePage/AdditionalMetadataFields.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+
+import AdditionalMetadataFields from './AdditionalMetadataFields';
+
+describe('AdditionalMetadata Fields', () => {
+  it('Display all fields', () => {
+    const component = shallow(<AdditionalMetadataFields />);
+    expect(shallowToJson(component)).toMatchSnapshot();
+  });
```

Comment:
huh, that's it?

Reply:
Yes actually this component is just adding fields, there is no any logic to test it for now IMO. May be when we will add bootcamp courses then we can add more test

Reply:
Ok.